### PR TITLE
HARP-4888: Refactor duplicated code in placeOverlayTextElements.

### PR DESCRIPTION
**Pull request by atomicsulfate**
Code was almost a 1-1 copy of initializeGlyphs() function.

Signed-off-by: Andres Mandado <andres.mandado-almajano@here.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: @here/harp-mapview/lib/text/TextElementsRenderer.ts [hunk 1]
```diff
@@ -737,7 +733,8 @@ export class TextElementsRenderer {
                 }
             }
 
-            if (!this.initializeGlyphs(textElement, textElementStyle)) {
+            const forceNewPassOnLoaded = true;
```
Comment (MPursche) [inline @here/harp-mapview/lib/text/TextElementsRenderer.ts]: Why is this a defined as a constant variable that is always true and not directly calling `!this.initializeGlyhps(textElement, textElementStyle, true)`?

File: @here/harp-mapview/lib/text/TextElementsRenderer.ts [hunk 2]
```diff
@@ -1310,48 +1306,9 @@ export class TextElementsRenderer {
                 continue;
             }
 
-            // Trigger the glyph load if needed.
-            if (textElement.loadingState === undefined) {
-                textElement.loadingState = LoadingState.Requested;
+            const forceNewPassOnLoaded = false;
```
Comment (MPursche) [inline @here/harp-mapview/lib/text/TextElementsRenderer.ts]: See above comment.

File: @here/harp-mapview/lib/text/TextElementsRenderer.ts (hunk 1 above)
Comment (atomicsulfate) [reply @here/harp-mapview/lib/text/TextElementsRenderer.ts]: It's just a way to self-document code: the constant name already tells you what the boolean does, if you just pass a boolean literal you need to look up the method documentation to see what the boolean means.

File: @here/harp-mapview/lib/text/TextElementsRenderer.ts (hunk 2 above)
Comment (atomicsulfate) [reply @here/harp-mapview/lib/text/TextElementsRenderer.ts]: idem.

File: @here/harp-mapview/lib/text/TextElementsRenderer.ts (hunk 1 above)
Comment (MPursche) [reply @here/harp-mapview/lib/text/TextElementsRenderer.ts]: I see.

**Pull request merged**